src/temp/ProjectListControls/ActiveSearchField.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState, ChangeEvent } from 'react';
+import { SearchTextField } from './SearchTextField';
+import CrossIcon from '../icons/CrossIcon';
+import style from './Search.module.scss';
+import SearchIcon from '../icons/SearchIcon';
+import { useTranslations } from 'next-intl';
+import { SetState } from '../../features/common/types/common';
+
+interface ActiveSearchFieldProps {
+  setIsSearching: SetState<boolean>;
+  setIsFilterOpen: SetState<boolean>;
+}
+const ActiveSearchField = ({
+  setIsSearching,
+  setIsFilterOpen,
+}: ActiveSearchFieldProps) => {
+  const [input, setInput] = useState('');
+  const t = useTranslations('AllProjects');
+
+  const resetSearchTab = () => {
+    setInput('');
+    setIsSearching(false);
+    setIsFilterOpen(false);
+  };
+
+  return (
+    <div className={style.searchBarMainContainer}>
```

Comment:
When this is open, there are 2 divs with the `searchBarMainContainer` class - this div and it's immediate parent from `ProjectListControls`